tools/crm_mon.c

```diff
@@ -601,6 +607,10 @@ static GOptionEntry display_entries[] = {
       "Show more details (node IDs, individual clone instances)",
       NULL },
 
+    { "show-description", 'd', G_OPTION_FLAG_NO_ARG, G_OPTION_ARG_CALLBACK, print_description_cb,
```

Comment:
'd' is already used. We can just do without a short option for this one (i.e. 0)